internal/provider/resources/routing_rules/routing_rule.go

```diff
@@ -80,6 +83,10 @@ See [the P0 request-routing docs](https://docs.p0.dev/just-in-time-access/reques
 					stringplanmodifier.RequiresReplace(),
 				},
 			},
+			"disabled": schema.BoolAttribute{
+				MarkdownDescription: "Whether or not the routing rule should be evaluated; if false or undefined, the rule will be evaluated",
```

Comment:
nit: "undefined" is a javascript term
```suggestion
				MarkdownDescription: "Whether or not the routing rule should be evaluated; if false or not specified, the rule will be evaluated",
```